_opensearch/snapshots/sm-dashboards.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: default
+title: Snapshot management in Dashboards
+parent: Snapshots
+nav_order: 35
+has_children: false
+---
+
+# Using snapshot management in Dashboards
+
+You can set up snapshot management (SM) in OpenSearch Dashboards.
+
+## Create a repository
+
+Before you create an SM policy, you need to set up a repository for snapshots. 
+
+1. In the top menu bar, go to **OpenSearch Plugins > Index Management**.
```

Comment:
@kolchfa-aws Users can also access Snapshots management from the top menu bar. Should this be the primary way to access the snapshot feature? 
![image](https://user-images.githubusercontent.com/3692324/177450873-dcd7a5b3-9364-473a-ab92-c33af89114af.png)

Reply:
Yes, it's probably easier to bypass Index Management. I will change the docs.